views/components/code_example/_code_example.scss

```diff
@@ -35,8 +35,19 @@
 	.output pre {
 		@extend %custom-prism-light-mode;		
 	}
+}
+figure {
+	margin: 5rem auto;
 
-	&--wide-margin {
-		margin: 5rem auto;
+	figcaption {
+		max-width: 40rem;
+		margin: 0.5rem auto;
```

Comment:
This looks a a little cramped to me on the landing page; a little more vertical spacing?